### ISSUE TITLE
DOC scrolling the left column menu scrolls the full page until bottom is reached, then scrolls the left menu

### ISSUE THREAD

**Issue by mathurinm** [scikit-learn-contrib/skglm#220]
In the screencast my mouse is on the left column and I'm scrolling down 

[Screencast from 10-01-2024 09:11:36.webm](https://github.com/scikit-learn-contrib/skglm/assets/8993218/4759394b-bbae-44b3-9e7e-5f72a221038f)

**Comment by Badr-MOUFAD** [comment]
I noticed many instabilities with the documentation theme we are using [(Furo theme)](https://github.com/pradyunsg/furo), for instance, the difference in links colors between stable and dev versions (as you highlighted last time @mathurinm)

I think we better switch to the theme used in benchopt [Book theme](sphinx-book-theme.readthedocs.io/en/stable/), it is more stable.

The switch should not be difficult as both themes share many similarities.

**Comment by mathurinm** [comment]
Agreed !  @PascalCarrivain do you feel like giving a try to migrating from the furo theme to the Book theme for our doc ?

**Comment by Badr-MOUFAD** [comment]
fixed thanks to #291
